pyproject.toml

```diff
@@ -8,17 +8,17 @@ readme = "README.md"
 packages = [{include = "balena"}]
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
+python = "^3.9.6"
```

Comment:
@JSReds i am not sure we can do this in a patch/if we can do this bump here. Basically we are deprecating support for python 3.6 3.7 and 3.8 in a patch bump. I am not sure we can do that as we have customers on python 3.7 (for sure) and I suspect some in 3.6